### DISCUSSION TITLE
Question about the article Layout Builder.

### DISCUSSION THREAD

**Discussion by patrickus33** [templaza/astroid-framework#773] [Q&A]
Hi,

Is it possible to insert a custom field in the article layout builder with shortcode like in my snapshot
![custom-field](https://github.com/user-attachments/assets/1d723f94-2cba-4333-9ee0-3e9687bc09da)
?
I use "Advanced Custom Fields"by Tassos

**Comment by paurray** [comment]
@patrickus33 have you seen this? Guess it is a similar use case?
https://github.com/templaza/astroid-framework/discussions/765

**Comment by patrickus33** [comment]
@paurray this is an article template made for one of my sites whit JCE pro.

**Comment by jam1974** [comment]
@patrickus33 https://github.com/templaza/astroid-framework/discussions/765#discussioncomment-10483163

**Comment by patrickus33** [comment]
@jam1974 Thank you for the idea!
Will try it!

**Reply by jam1974** [reply]
No modules can yet be loaded into the Article Layout Builder
